README.md

```diff
@@ -66,6 +66,14 @@ function animate() {
 
 If everything went well, you should see [this](https://jsfiddle.net/8kubjpL5/).
 
+### Cloning this repository ###
+
+To avoid cloning all the history this command will reduce the download size in half.
```

Comment:
I think the description is a bit confusing, how do you feel about the following?
>Use `--depth` parameter to prevent cloning all commit history. It will significantly reduce the download size."

Reply:
Sounds good!

Reply:
We also can explain anything here and just suggest with `—depth`, curious developers will understand anyway. Also we can place explanation here https://github.com/mrdoob/three.js/wiki/Build-instructions

Reply:
> Use --depth parameter to prevent cloning all commit history. It will significantly reduce the download size."

Minor grammar nits: _Use **the** `--depth` parameter to prevent cloning **the entire** commit history. **This** will significantly reduce the download size."_